index.rst

```diff
@@ -13,17 +13,312 @@
+so the baseline would be to make all DP0.2 and DP0.3 data accessible to the same set of users
+and base it on the CILogon/COmanage IAM mechanism to which DP0.2 is in the process of being transitioned.
+
+
+Timeline
+========
+
+- January 2023: Refine definition of this Data Preview.
+  Review existing SSSC data model for any changes needed for DPDD and/or standards (e.g., IVOA)
+  conformance (Dubois-Felsmann, Juric).
+
+- February 2023: Regenerate simulation (Juric).
+  Produce initial Felis data model (Dubois-Felsmann).
+  Prepare necessary USDF infrastructure (R. Dubois).
+
+- March 2023: Initial version of data turned over to USDF team.
+  Ingest into Postgres at USDF (Mueller).
+  Fix/regenerate source data if problems are found.
+
+- April 2023: Establish TAP service in Google Cloud over USDF Postgres DB (SQuaRE).
+  Complete Portal (context image selection) and Notebook (TAP service helper function) software refinements.
+
+- May 2023: Exercise DP0.3 internally.
+  Complete DataLink microservices and metadata deployment (SQuaRE and D-F).
+  CET develops tutorials and notebooks.
+
+- 26 May 2023: Release candidate turned over to product owners.
+
+- 13 June 2023: Nominal release date.
+
+- June-September 2023: Public (RTN-011) commitment for release date.
+
+
+Preparations Required
+=====================
+
+Regeneration of the Simulation
+------------------------------
+
+An initial review of the existing SSSC simulation, performed last year by Gregory Dubois-Felsmann
+and Mario Juric, exposed some minor issues in consistency with the DPDD and other aspects of the
+schema.
+Mario described the changes that were suggested as very easy to make.
+
+We also understand that the SSSC will in any event want to regenerate the simulation based on
+a more recent survey plan simulation.
+
+So the first step in preparation for DP0.3 would be to confirm the changes needed and then
+proceed to re-run the simulation.
+If a visit table is decided to be a useful adjunct to DP0.3, this would be the time to define
+and generate it.
+
+Database Setup
+--------------
+
+Ingest
+------
+
+On `PREOPS-1152`_, Mario Juric reports that:
+
+"For our internal use, we've used pg_bulkload to rapidly (in ~30 minutes) ingest these tables into a database.
+The details are in this (messy) notebook.
+Using more typical loading mechanisms (from .csv files, etc.) is not an issue, just will be slower.
+
+"If a postgres database can be set up within the RSP, with pg_bulkload enabled and given administrative permissions I would be able to load these data into it probably in a ~few days.
+This setup would also allow for uploads of future dataset updates: we refresh these simulations ~annually, as new baseline simulations become available and the software is improved."
+
+Data Model Metadata
+-------------------
+
+Service Deployment
+------------------
+
+Potential New Services
+^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
What is the ETA for filling these section in? We could release a first version now without this but if so, add a line that these sections will be completed once we have a better idea of the implementation details